change-beta/@azure-communication-react-12a92b59-bacd-414a-927d-fe69b009f6cf.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Added maxPinnedRemoteVideoTiles prop to VideoGallery. Disabling pin menu button on remote VideoTile of VideoGallery when max pinned remote video tiles is reached",
```

Comment:
As we are not adding the `maxPinnedRemoteVideoTiles ` prop, the comment seems misleading.